src/editor.js

```diff
@@ -1777,6 +1782,11 @@ export class OdooEditor extends EventTarget {
         ev.preventDefault();
         const pastedText = (ev.originalEvent || ev).clipboardData.getData('text/plain');
         const splitAroundUrl = pastedText.split(URL_REGEX);
+        const htmlParams =
+            Object.keys(this.options.addedLinkHtmlParameters)
+                .map(key => [key, this.options.addedLinkHtmlParameters[key]].join('="'))
```

Comment:
Again splitting hairs:
Why not use `Object.entries` here too?
```js
const htmlParams = Object.entries(this.options.addedLinkHtmlParameters)
    .map(entry => entry.join('="')).join('" ') + '"';
```

Reply:
:+1: